static/app/views/issueDetails/streamline/eventMissingBanner.tsx

```diff
@@ -29,7 +30,10 @@ export function EventMissingBanner() {
       'Switch over to a [link:recommended] event instead, it should have more useful data.',
       {
         link: (
-          <Link to={`${baseUrl}/recommended/`} aria-label={t('View recommended event')} />
+          <Link
+            to={`${baseUrl}/recommended/statsPeriod=${MAX_PICKABLE_DAYS}d`}
```

Comment:
i think we should preserve the projectId query parameter if present on the links here or add it to the links